{"url":"https://api.github.com/repos/owncloud/core/issues/4598","labels_url":"https://api.github.com/repos/owncloud/core/issues/4598/labels{/name}","comments_url":"https://api.github.com/repos/owncloud/core/issues/4598/comments","events_url":"https://api.github.com/repos/owncloud/core/issues/4598/events","html_url":"https://github.com/owncloud/core/issues/4598","id":18608977,"number":4598,"title":"webdav server dropping connections/unstable. ","user":{"login":"kudeta","id":2739450,"avatar_url":"https://gravatar.com/avatar/f75c8d10a13ab055320de12be7e4f583?d=https%3a%2f%2fidenticons.github.com%2f613d110e11784be45f0e29dbacf7d4f5.png&r=x","gravatar_id":"f75c8d10a13ab055320de12be7e4f583","url":"https://api.github.com/users/kudeta","html_url":"https://github.com/kudeta","followers_url":"https://api.github.com/users/kudeta/followers","following_url":"https://api.github.com/users/kudeta/following{/other_user}","gists_url":"https://api.github.com/users/kudeta/gists{/gist_id}","starred_url":"https://api.github.com/users/kudeta/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/kudeta/subscriptions","organizations_url":"https://api.github.com/users/kudeta/orgs","repos_url":"https://api.github.com/users/kudeta/repos","events_url":"https://api.github.com/users/kudeta/events{/privacy}","received_events_url":"https://api.github.com/users/kudeta/received_events","type":"user","site_admin":false},"labels":[{"url":"https://api.github.com/repos/owncloud/core/labels/bug","name":"bug","color":"fc2929"}],"state":"open","assignee":null,"milestone":null,"comments":3,"created_at":"2013-08-27t13:59:01z","updated_at":"2013-11-29t11:55:47z","closed_at":null,"pull_request":{"html_url":null,"diff_url":null,"patch_url":null},"body":"hiya,\r\n\r\nmy first ever bug report so i hope i'm writing this in the correct place.\r\n\r\nbackground: i've been struggling for some time to sync my favorite information management program, devonthink, with the webdav server provided by owncloud. (i've been trying since 4.5.xx). thus far i had given up and resorted to including the database file within the sync client, that has worked just fine.\r\n\r\ni'm getting errors from devonthink such as:\r\n\r\n27/08/2013 14:16:34.642 devonthink pro[81897]: fetch item records/69 existence error: error domain=nsurlerrordomain code=-1001 \"connection failed because the connection timed out.\" userinfo=0x5dbfd0 {nslocalizeddescription=connection failed because the connection timed out.}\r\n\r\nit just keeps timing out.\r\n\r\ni thought this was an issue with devonthink, but recently tried mounting the webdav share on the same, osx machine and get very bad results:\r\n\r\n1. mount the owncloud share\r\n2. attempt to upload a file, to which i see \"file already exists - replace file??\" - note the file does most definitely not exist\r\n3. the file appears to upload just fine. \r\n4. at the of the upload i get an os error -36 or -48 (but the file appears to have completed upload just fine, and can be seen in my owncloud interface)\r\n5. the owncloud mount then often immediately drops.\r\n\r\nsometimes i can download files from the owncloud server over webdav sync, sometimes it also refuses.\r\n\r\nowncloud tells me:\r\n\r\n\"\"iswebdavworking: no - reason: exception 'sabre_dav_exception' with message '[curl] error while making request: couldn't connect to host (error code: 7)' in /var/www/owncloud/3rdparty/sabre/dav/client.php:410 stack trace: #0 /var/www/owncloud/3rdparty/sabre/dav/client.php(179): sabre_dav_client->request('propfind', '', '<?xml version=\"...', array) #1 /var/www/owncloud/lib/util.php(609): sabre_dav_client->propfind('', array) #2 /var/www/owncloud/settings/admin.php(34): oc_util::iswebdavworking() #3 /var/www/owncloud/lib/route.php(113) : runtime-created function(1): require_once('/var/www/ownclo...') #4 [internal function]: __lambda_func(array) #5 /var/www/owncloud/lib/router.php(127): call_user_func('?lambda_8', array) #6 /var/www/owncloud/lib/base.php(620): oc_router->match('/settings/admin') #7 /var/www/owncloud/index.php(28): oc::handlerequest() #8 {main} \"\"\r\n\r\ni see a message on the admin page that webdav appears to be borked (your web server is not yet properly setup to allow files synchronization because the webdav interface seems to be broken. )\r\n\r\ni'm also seeing lots of: filesize(): stat failed for /var/www/owncloud/....\r\n\r\ni see nothing pertinent in the apache logs.\r\n\r\n\r\nthe server is a vm (ubuntu 11.10) running on my personal dedicated remote host. php etc is up to date. i'm not running any other webdav servers. i'm running the latest available release of owncloud and to reiterate, i'm not having any trouble using the sync client that owncloud provide.\r\n\r\n\r\n\r\nplease let me know if there is anything else i can provide to help diagnose!\r\n\r\nthanks,\r\n\r\nowncloud is great!\r\n\r\n","closed_by":null}